modules/sdk-core/src/bitgo/wallet/iWallet.ts

```diff
@@ -599,6 +609,7 @@ export interface IWallet {
   updateForwarders(forwarderFlags?: any): Promise<any>;
   deployForwarders(params: DeployForwardersOptions): Promise<any>;
   flushForwarderToken(params: FlushForwarderTokenOptions): Promise<any>;
+  getForwarderBalance(params: ForwarderBalanceOptions): Promise<ForwarderBalance[]>;
```

Comment:
seems this param is optional in the implementation
```suggestion
  getForwarderBalance(params?: ForwarderBalanceOptions): Promise<ForwarderBalance[]>;
```